internal/provider/provider.go

```diff
@@ -262,12 +262,12 @@ func (p Provider) Schema(ctx context.Context, request provider.SchemaRequest, re
 			// OIDC specific fields
 			"oidc_request_token": schema.StringAttribute{
 				Optional:            true,
-				MarkdownDescription: "The bearer token for the request to the OIDC provider. This can also be sourced from the `ARM_OIDC_REQUEST_TOKEN` or `ACTIONS_ID_TOKEN_REQUEST_TOKEN` Environment Variables.",
+				MarkdownDescription: "The bearer token for the request to the OIDC provider. This can also be sourced from the `ARM_OIDC_REQUEST_TOKEN`, `ACTIONS_ID_TOKEN_REQUEST_TOKEN`, or `SYSTEM_ACCESSTOKEN` Environment Variables.",
 			},
 
 			"oidc_request_url": schema.StringAttribute{
 				Optional:            true,
-				MarkdownDescription: "The URL for the OIDC provider from which to request an ID token. This can also be sourced from the `ARM_OIDC_REQUEST_URL` or `ACTIONS_ID_TOKEN_REQUEST_URL` Environment Variables.",
+				MarkdownDescription: "The URL for the OIDC provider from which to request an ID token. This can also be sourced from the `ARM_OIDC_REQUEST_URL`, `ACTIONS_ID_TOKEN_REQUEST_URL`, or `SYSTEM_OIDCREQUESTURI` Environment Variables.",
```

Comment:
Please also update below source code to get the value from `SYSTEM_OIDCREQUESTURI`

https://github.com/Azure/terraform-provider-azapi/pull/852/files#diff-58d6a027753b50994deb7e11e4a99dde423f35844986019bd9cea5e0c94aba22L507

Reply:
It is hard-coded here, so removed it from this code. I had added it origianlly, but wouldn't be used: https://github.com/Azure/azure-sdk-for-go/blob/078e06aa73fb6a8dc585ad4ce1b371780aef2237/sdk/azidentity/azure_pipelines_credential.go#L22

Reply:
Thanks for the clarification!